lib/script/script.js

```diff
@@ -980,13 +1055,17 @@ Script.prototype.toScriptHashOut = function() {
 
 /**
  * @return {Script} an output script built from the address
+ * @param {(Address|PublicKey)} to - destination address or public key
+ * @param {(string)} coldStakingAddress - cold staking address
  */
-Script.fromAddress = function(address) {
+Script.fromAddress = function(address, coldStakingAddress) {
   address = Address(address);
   if (address.isPayToScriptHash()) {
     return Script.buildScriptHashOut(address);
   } else if (address.isPayToPublicKeyHash()) {
     return Script.buildPublicKeyHashOut(address);
+  } else if (address.isPayToPublicKeyHash256()) {
```

Comment:
Some day you'll want to add the script hash version to support multisig.

Reply:
Correct, i have another task to add the multi sig staking